packages/reakit/src/_utils/pickHTMLProps.ts

```diff
@@ -1,21 +1,14 @@
-import { getElementProps, getEventProps } from "react-known-props";
-
-const pickHTMLProps = (tagName: string, props: { [key: string]: any }) => {
-  const allowedProps = [
-    ...getElementProps(tagName, { legacy: true }),
-    ...getEventProps()
-  ];
-
-  const testProp = (prop: string) =>
-    allowedProps.indexOf(prop) >= 0 || /^data-.+/.test(prop);
-
-  return Object.keys(props).reduce(
-    (finalProps, key) => ({
-      ...finalProps,
-      ...(testProp(key) ? { [key]: props[key] } : {})
-    }),
-    {}
-  );
-};
+import validAttr from "styled-components/lib/utils/validAttr";
```

Comment:
this is not quite reusing anything (bytes-wise) - I'm not even sure which task generates `lib` directory in `styled-components`, but they flat bundle their main dist targets so deep paths like this are not supported - people using modern bundlers will import an [esm flat bundle](https://github.com/styled-components/styled-components/blob/f401eda35ef16065509ea8a7375290fb8b5858a6/package.json#L8) which already contains this `validAttr` in it, so importing it here from `/lib/utils/validAtts` will load the duplicate of this code

Reply:
Well, I was hoping that gzip could do its job.

Reply:
You're right. I can't find anything on the styled-components code that generates `lib`. It may be a manual babel cli call.

Will find another way to import it.

Reply:
So, basically, (I'm trying to understand this) using `styled-components/lib/utils/validAttr` is not good because